portfolio-main.css

```diff
@@ -0,0 +1,187 @@
+
+/* GLOBAL STYLES */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Status bar and navigation bar style */
+.nav-bar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
+
+.status-bar {
+  display: flex;
+  justify-content: space-between;
+  position: absolute;
+  width: 375px;
+  height: 44px;
+  left: calc(50% - 375px / 2);
+  top: 0;
+}
+
+.status-bar p {
+  position: absolute;
+  width: 54px;
+  height: 16px;
+  left: 0;
+  top: calc(50% - 16px / 2 + 0.5px);
+  font-weight: 500;
+  font-size: 14px;
+  letter-spacing: -0.28px;
+}
+
+.toolbar p {
+  width: 137px;
+  padding: 15px;
+  font-weight: 600;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.status-bar p,
+.toolbar p {
+  font-family: Roboto, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
- [ ] Great job adding the fonts from the Figma design 🌵, but when you deploy your project, the fonts will change, please add in the `<head>` of your `.html` file the CDN link, 🖼️  I suggest you use [Google Fonts.](https://fonts.google.com/). Also, you can check the rendered fonts if you inspect your project in the browser, press the `computed` tab at the end of it you will find the rendered font. 💯 

![image](https://user-images.githubusercontent.com/91301423/198421045-9bace145-822b-478a-813c-1f1cab9d7713.png)